include/SugarObjects/templates/person/vardefs.php

```diff
@@ -253,7 +253,46 @@
             'massupdate' => false,
             'studio' => 'false',
         ),
-
+        'lawful_basis' => array(
+            'name' => 'lawful_basis',
+            'vname' => 'LBL_LAWFUL_BASIS',
+            'type' => 'multienum',
+            'massupdate' => true,
+            'no_default' => false,
+            'comments' => '',
+            'inline_edit' => true,
+            'reportable' => true,
+            'merge_filter' => 'enabled',
+            'len' => 100,
+            'size' => '20',
+            'options' => 'lawful_basiss',
```

Comment:
same here.... there is no need to identify it as lawful_basis... Would it be better to set lawful_basis_dom like other standard/core modules?